tests/test_modelscan.py

```diff
@@ -463,16 +463,26 @@ def test_scan_zip(zip_file_path: Any) -> None:
 def test_scan_pytorch(pytorch_file_path: Any) -> None:
     ms = ModelScan()
     results = ms.scan(Path(f"{pytorch_file_path}/bad_pytorch.pt"))
-    assert results["summary"]["scanned"]["scanned_files"] == [f"bad_pytorch.pt"]
-    assert results["summary"]["skipped"]["skipped_files"] == []
+
+    assert results["errors"] == [
```

Comment:
Thinking this should be a skipped reason instead of an error. In the benchmarking we're encountering .bin files that aren't pytorch and we should just be skipping them instead of erroring